Specs/Scene/Cesium3DTilesetSpec.js

```diff
@@ -4938,7 +4938,9 @@ describe(
         scene,
         gltfContentWithCopyrightUrl
       ).then(function (tileset) {
-        viewGltfContent();
+        setZoom(10.0);
+        scene.renderForSpecs();
```

Comment:
Does this fix something from the previous PR?

Reply:
Sort of -- I found that when creating the other tests that the child tiles didn't show up with the zoom specified in `viewGltfContent()`. But I forgot that `scene.renderForSpecs()` is called in ``waitForTilesLoaded`, so wherever the camera is when that is called is fine. Should I omit the camera movement altogether in this test?

Reply:
No it's fine. I was just curious.